explanation/what-is-modeled.md

```diff
@@ -0,0 +1,41 @@
+# What is Modeled 
+````{margin}
+```{note}
+Borrowing the term entity from brick
+```
+````
+This standard allows users to create semantic models that describe the many entities in their buildings that are relevant to the use cases described in (the landing page)[landing-page]. An entity is an abstraction of any physical, logical or virtual item; the actual "things" in a building. Examples are mechanical equipment such as air handling units, luminaires, spatial elements like rooms, the area of these rooms served by certain mechanical equipment, or the zones these rooms may be grouped into based on how the equipment is controlled.
+
+There is a huge amount of information about the many entities within a building that you may want to model. This section will explain what information you should represent in a 223P model and the 223P modeling constructs you will use to do so. Descriptions assume you have some familiarity with RDF graph data and know the [definitions of key terms used in this section](definitions). 
+
+## Type
+
+This standard provides well defined classes used to describe entities relevant to building system information. Entities in s223 models will generally be instances of the many classes defined or referred to by the standard. The classes in the standard provide 'names' for the fundamental building blocks used in 223P models (e.g. a fan will be an instance of the class [s223:Fan](https://explore.open223.info/s223/Fan.html)) and also have rules defining how they are used (e.g. a fan must convey air).
+
+## Topology
+
+This standard can be used to describe the topology of the equipment and spaces in a building, but not the geometric details. Topology refers to the way entities are connected and how some media (e.g. water, air, or electricity) is conveyed between them. There are several different classes used to describe which entities participate in connections and how they connect. This describe [Connectables](https://explore.open223.info/s223/Connectable.html), which include the entities that are capable of connecting to each other; [ConnectionPoints](https://explore.open223.info/s223/ConnectionPoint.html), which are the points where Connectables are connected; and [Connections](https://explore.open223.info/s223/Connection.html), which describe physical things through which the medium is conveyed, like pipes or ducts. 
```

Comment:
ConnectionPoints definition change to "which model where Connectables can be connected"